rosdep/python.yaml

```diff
@@ -3358,6 +3358,16 @@ python-pexpect:
   debian: [python-pexpect]
   gentoo: [dev-python/pexpect]
   ubuntu: [python-pexpect]
+python-pickleDB:
```

Comment:
Since this is a pip-only key, please make this `python-pickleDB-pip`.  We may also want to add `python3-pickleDB-pip` for Python3.

Reply:
changed name to python-pickleDB-pip